test/CMakeLists.txt

```diff
@@ -15,3 +15,7 @@ tdl_fetch_google_test_test ()
 add_tdl_test(ParamValue_test.cpp)
 add_tdl_test(Param_test.cpp)
 add_tdl_test(ParamCTDFile_test.cpp)
+
+add_executable (test_symbol_collision collision1.cpp collision2.cpp)
```

Comment:
Maybe call them collision_test1/2.cpp :nail_care: ?

Reply:
renamed them to "multiple_definition01_test.cpp". Also added a short description to the first file to make clear, what they are all about.